### ISSUE TITLE
Improve prefer-immediate-return: keep comments when fixing

### ISSUE THREAD

**Issue by amankkg** [SonarSource/eslint-plugin-sonarjs#198]
**I want to report a bug.**

**Reproducer**

```typescript
// minimal reproducer if relevant
const format = (n: string) => {
  // const n = digits(string)
  // const fmt = new l.AsYouType()
  const input = fmt.input(n)

  // if (!input) return 'foo'

  return input
}
```

**Expected behavior**

Rule fix is applied. Commented blocks remain inside a function.

**Actual behavior**
```typescript
const format = (n: string) => {
  // const n = digits(string)
  // const fmt = new l.AsYouType()
  return fmt.input(n)
}
```

**eslint-plugin-sonarjs version:** 0.5.0<br>
**eslint version:** 6.8.0<br>
**Node.js version:** 12<br>

**Rule key:** sonarjs/prefer-immediate-return
